### PR TITLE
make guilds not display group plans as title section

### LINKED ISSUE #12738: HTML page Title tag for Guilds says "Group Plans"
When you are viewing a Guild that does not have a Group Plan, the title says "Group Plans".
It should say "Guilds".

The two examples below are public guilds that definitely don't have Group Plans in them.

image
image

_Thanks to @citrusella for reporting this._

### PR DESCRIPTION
**Pull request by thewilloftheshadow**
Fixes #12738

### Changes
Added code to detect if the page was Group Plans or Guilds and set the title appropriately 

#### One thing that should be checked: 
The Task Board page of a group plan has `Test 1 | Group | Habitica`, while the Group Information page now has `Test 1 | Group Plans | Habitica`

----
UUID: 2eb203f5-1b1e-4e5d-84a4-22f130a1090e

### CODE REVIEW

File: website/client/src/components/groups/group.vue [hunk 1]
```diff
@@ -584,7 +584,6 @@ export default {
         this.$root.$emit('bv::show::modal', 'create-party-modal');
         return;
       }
-
       if (this.isParty) {
```
Comment (paglias) [inline website/client/src/components/groups/group.vue:590]: @thewilloftheshadow should this if block get fixed as well? Parties can be group plans as well

**Review by paglias — CHANGES_REQUESTED** [review]
@thewilloftheshadow thanks! I've left one comment.

Regarding

> The Task Board page of a group plan has Test 1 | Group | Habitica, while the Group Information page now has Test 1 | Group Plans | Habitica

I'd change everything to `Group Plans`

File: website/client/src/components/groups/group.vue (hunk 1 above)
Comment (thewilloftheshadow) [reply website/client/src/components/groups/group.vue:590]: Ooo I didn't know there were parties that had group plans, yea I'll fix that too

**Comment by paglias** [comment]
thanks @thewilloftheshadow , I fixed a small bug with parties titles and merged the PR. Noted towards your 2nd tier

**Pull request merged**